packages/generator/src/edmx-to-vdm/v4/operation.ts

```diff
@@ -20,95 +29,280 @@ const logger = createLogger({
 
 const extractResponse = (response: string) => `${response}.value`;
 
-interface JoinedOperationData {
-  operationImport: EdmxOperationImport;
-  operation: EdmxOperation;
+function splitMissingOperation(
+  operationImports: EdmxOperationImport[],
+  operations: EdmxOperation[]
+): [EdmxOperationImport[], EdmxJoinedOperation[]] {
+  return operationImports.reduce<
```

Comment:
[req] didn't you introduce a `partition` or `divide` method on arrays? If so please use this here or create a follow up to use this for the two methods here.

Reply:
`partition` already existed in `array.ts` but it can't be easily used here because we have 2 type parameters for the 2 result lists with transformation happening inside the reduce.. maybe we can split the transformation and the partition in 2 steps?

Reply:
no I missed the point of the two data points.